plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizardFragment.java

```diff
@@ -521,4 +526,9 @@ public void setComplete(boolean complete) {
 		super.setComplete(complete);
 	}
 
+	@Override
+	public boolean isComplete() {
+		return uiHook != null && !uiHook.isDisposed() && super.isComplete();
+	}
```

Comment:
If I understand the logic right we can only have this fragment completed if it created its composite.
Why not simply (instead of keeping a reference to some child widget) use #getControl() (#getControl() != null && !#getControl().isDisposed() && super.isComplete())?